### PR TITLE
Version 1 changes

### PR DESCRIPTION
**Pull request by shahnozabonu87**
Hi @shahnozabonu87 

Created a test cases for login feature , kindly check the PR and approve

### CODE REVIEW

File: Demo.java [hunk 1]
```diff
@@ -0,0 +1,6 @@
+public class Demo {
+
+    public static void main(String[] args) {
+        System.out.println("Version 1");
```
Comment (Riyaz-commits) [inline Demo.java:4]: please us eth existing code

Comment (shahnozabonu87) [reply Demo.java:4]: Changes were made as suggested.

Comment (Riyaz-commits) [reply Demo.java:4]: LGTM

**Review by Riyaz-commits — APPROVED** [review]
LGTM

**Pull request merged**